### DISCUSSION TITLE
Custom map without OpenDrive

### DISCUSSION THREAD

**Discussion by drbswa** [carla-simulator/carla#4712] [Ideas]
I think it would be nice to add the ability to add custom maps without requiring OpenMap data. It would open up the use cases to things like off-road driving, which has had an increase in interest lately.

**Comment by sguttikon** [comment]
As workaround, i create an empty openmap data. But I also had to get rid of some functionalities such as traffic manager.
https://github.com/carla-simulator/carla/issues/3623#issuecomment-791586976

<custom_map>/OpenDrive/<custom_map>.xodr content:
```
<?xml version="1.0" encoding="utf-8"?>
<OpenDRIVE/>
```
